libs/blocks/global-navigation/global-navigation.css

```diff
@@ -569,6 +568,11 @@ header.global-navigation {
     padding: 40px 0 0;
     max-height: calc(100vh - 100%);
     overflow: auto;
+    box-sizing: border-box;
```

Comment:
I don't see any box-size model properties adapted to justify this change. Why is it needed?

Reply:
I think this was needed to compensate the 40px padding which was not included in height and the bottom-most link in the large menu was getting cut off from the scrolling as shown.

<img width="1337" alt="Screenshot 2024-07-24 at 1 29 19 PM" src="https://github.com/user-attachments/assets/11ee6c42-5839-4df3-8c9d-74ef6da06d2b">